lib/api/core/plugins/pluginsManager.js

```diff
@@ -1063,6 +1065,32 @@ class PluginsManager {
             : {}
         };
 
+      // load customs errors configuration file
+      if (!plugin.manifest.raw.errors) {
+        this.kuzzle.log.warn(`[${plugin.manifest.name}] This plugin doesn't have custom errors in the manifest file.`);
+      } else {
+        try {
+          const pluginCode = this.config[plugin.manifest.name] && this.config[plugin.manifest.name].code
+            ? this.config[plugin.manifest.name].code
+            : 0x00;
+          loadPluginsErrors(plugin.manifest.raw, pluginCode);
+          this.kuzzle.log.info(`[${plugin.manifest.name}] Custom errors successfully loaded.`);
+        } catch (e) {
+          if (e.message.match(/Error configuration file/i)
+              || e instanceof SyntaxError
+          ) {
+            errorsManager.throw(
+              'plugins',
+              'runtime',
+              'errors_configuration_file',
+              plugin.manifest.name,
+              e.message);
+          } else {
+            this.kuzzle.log.warn(`[${plugin.manifest.name}] ${e}.`);
```

Comment:
... I don't get why we would log a mere warning when a non-KuzzleError error is thrown. 

If any error is thrown during the parsing of a manifest.json file, we should throw an error to abort Kuzzle's start sequence.

Reply:
Above, it throws if the json file is bad formatted.
And if the json is not the problem, that means that we can still use it (for other errors from plugins).
And because this is an optional feature, I thought it was better just to warn instead of aborting